libs/sdk-ui-charts/src/charts/headline/Headline.tsx

```diff
@@ -30,6 +30,12 @@ export interface IHeadlineBucketProps {
      * Optionally specify filters to apply on the data to chart.
      */
     filters?: INullableFilter[];
+
+    /**
+     * Update headline items height according the specified size selected when its value is set to true.
```

Comment:
Comment should mention, that it allows more compact render of Headline, when its container/size is too small